packages/terra-popup/tests/nightwatch/popup-spec.js

```diff
@@ -0,0 +1,233 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const screenshot = require('terra-toolkit').screenshot;
+
+module.exports = {
+  before: (browser, done) => {
+    browser.resizeWindow(browser.globals.width, browser.globals.height, done);
+  },
+
+  afterEach: (browser, done) => {
+    screenshot(browser, 'terra-Popup', done);
+  },
+
+  'isOpen prop - RESIZE': (browser) => {
+    browser
+      .url(`http://localhost:${browser.globals.webpackDevServerPort}/#/tests/popup-tests/default`)
+      .waitForElementPresent('#default-button', 1000)
```

Comment:
While going through the css module conversions, we have been removing waitForElementPresent because its causing our nightwatch tests to take significantly longer waiting a second for each assertion.